src/agent/cmd/main.go

```diff
@@ -61,30 +61,36 @@ func main() {
 	}
 
 	deployer := deployer.New(*apiServerAddr, cfg.nodeName, cfg.podID)
-	err = retry.ExpBackOffWithLimit(func() error {
+	for {
+		communicateWithNode(cfg.nodeName, deployer)
```

Comment:
we need to capture `communicateWithNode` return error msg and then log to `log.Errorf`

Reply:
done